### ISSUE TITLE
RPR: Enshroud Combo

### ISSUE THREAD

**Issue by TheFallenZX** [Nik-Potokar/XIVSlothCombo#1566]
Messed with a few diffrent setting options, when using the "Enshroud Combo" under "Gluttony on Blood Stalk ect"
Does not add the oGCD Lemure's Slice to the combo, it is only using the main 2 GCD options.

Set up to change Stalk into Gluttony, which was also changing it to the full combo.

only options enabled on Reaper.
![Untitled](https://github.com/user-attachments/assets/d9fdead9-b72e-455e-bd51-bdb38e524f8a)
![image](https://github.com/user-attachments/assets/89f874f4-b6b2-45db-bd5f-619fc92b69dd)

**Comment by Kagekazu** [comment]
Duplicate of #1556

**Comment by TheFallenZX** [comment]
> Duplicate of #1556

I did see that, and i did see that it was stated "Fixed"
and an update https://github.com/Nik-Potokar/XIVSlothCombo/pull/1532
shows #1556 fixed, but it is still not working.

unless i am understanding the fix/patch fix thing wrong.

**Comment by Kagekazu** [comment]
Cause it's not been pushed to live. It will be fixed once the PR is merged.

**Comment by TheFallenZX** [comment]
> Cause it's not been pushed to live. It will be fixed once the PR is merged.

Understood!
failure on my part not understanding how that bug fixes page reads out!

**Comment by Kagekazu** [comment]
> > Cause it's not been pushed to live. It will be fixed once the PR is merged.
> 
> 
> 
> Understood!
> 
> failure on my part not understanding how that bug fixes page reads out!

No worries!
